frontend/apps/web/components/jobs/SchemaTable/FormErrorsCard.tsx

```diff
@@ -109,6 +128,9 @@ export default function FormErrorsCard(props: Props): ReactElement {
 function formErrorsToMessages(errors: FormError[]): string[] {
```

Comment:
I'm very performance sensitive about things changing on the schema table page. 
There are potentially a lot of form errors that can occur on this page...

It might be more efficient here to have a `formErrorsToMessages` function return an object that splits them.
```
interface FormErrorMesageResponse {
  errors: string[];
  warnings: string[];
}
```
Then loop through them and check the level.
That way you reduce the algorithm from O(2N) to just O(N)